JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -1,9 +1,59 @@
-﻿module Microsoft.ApplicationInsights {
+﻿/// <reference path="./logging.ts" />
+module Microsoft.ApplicationInsights {
 
     export class Util {
         private static document: any = typeof document !== "undefined" ? document : {};
-
         public static NotSpecified = "not_specified";
+
+        /*
+         * helper methods to access local storage
+         */
+        public static _getStorageObject():Storage {
```

Comment:
This is accessed on line 172 in the renew function in Session.ts. It's used to directly test if storage is available so we can fire an internal log if it's not. That way we can check in kusto what browsers+versions and how many users are unable to use local storage.